src/shared/utils/authStorage.ts

```diff
@@ -0,0 +1,27 @@
+import { Workspace } from 'shared/state';
+
+import { LocalStorageKeys } from './storage';
+
+const set = (key: string, data: unknown) => sessionStorage.setItem(key, JSON.stringify(data));
+const get = (key: string) => {
+  const data = sessionStorage.getItem(key);
+
+  try {
+    return data ? JSON.parse(data) : null;
+  } catch {
+    return data;
+  }
+};
+const remove = (key: string) => sessionStorage.removeItem(key);
+
+export const authStorage = {
+  getRefreshToken: () => get(LocalStorageKeys.RefreshToken),
```

Comment:
Looks incorrect to use LocalStorage keys in session storage, maybe devide them into two separate enums?

Reply:
fixed